juriscraper/pacer/email.py

```diff
@@ -566,6 +575,17 @@ def _parse_bankruptcy_short_description(self, subject: str) -> str:
             # Out: Reply
             short_description = subject.split(case_name)[-1]
 
+        else:
+            logger.error(
+                "Short description has no parsing for bankruptcy court '%s'",
```

Comment:
Also, it would be good to use an f-string here.

Reply:
Do we have a strict rule about this, or do we follow pylint?

On my editor pylint suggests using "%s" interpolation for logging functions

See:
https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html
and related [SO discussion](https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation)

Reply:
Yeah, you're right. I think this is also why Alberto's log error was showing up in Sentry a gazillion times. IIRC, Sentry can group logged errors if you use `%s`, but not if you use other methods.